### DISCUSSION TITLE
Who is your "middle man" monitoring provider?

### DISCUSSION THREAD

**Discussion by TheWebMachine** [pyalarmdotcom/alarmdotcom#374] [Polls]
While we've all been hunting and pecking at the state change issues and such, something occured to me: Some monitoring companies integrate with Alarm-dot-com to different degrees and in different ways, even if they are all using the ADC cloud for the main backend.

For example, ADT appears to loosely layer over ADC and just passes everything directly to ADC, resulting in a mostly "pure" ADC experience and compatibility with this integration. However, others work rather differently and this could be affecting reporting and attempts at replicating issues. 

I couldn't possibly name all the companies that partner with ADC (I listed the most popular US companies) - which is kinda bad because if ADC goes down or gets hacked, over 100 companies (I stopped counting) could all suffer at once for it, which is a LOT of eggs in one basket - but it might help to know the most popular ones and determine to what extent they interoperate with ADC.

It also might help if we all mention which middle man we're using when submitting issues in the future to ensure no one is chasing a "bug" that is really a limitation of one particular partner of ADC.

So, it begs the question...

**Comment by Nealtron** [comment]
SafeTouch Security.

**Comment by timinski** [comment]
Wilson's Security

**Comment by ronald-mendoza** [comment]
Alert 360 (used to be My Alarm Center partnered with Frontpoint for customer support)

**Comment by andyrozman** [comment]
Action24 (Ireland)
